tests/third_party/cupy/testing/helper.py

```diff
@@ -1005,6 +1005,12 @@ def decorator(impl):
         @_wraps_partial(impl, name)
         def test_func(*args, **kw):
             for dtype in dtypes:
+                if (
+                    dtype in ["d", "D"]
```

Comment:
probably it would be more correct to have something more common, to cover all use case, like:
```suggestion
                    numpy.dtype(dtype).kind in ("d", "D")
```